www/userguide/userguide.md

```diff
@@ -299,6 +311,18 @@ We are mapping checker names to different severity levels:
 (E.g. _core.DivideZero, core.NullDereference, cplusplus.NewDelete, etc._)
 - <span class="customIcon icon-severity-critical"></span> **Critical**
 
+## <a name="userguide-manage-source-components"></a> Manage source components
+Source components are named collection of directories specified as directory
```

Comment:
What do you think about rephrasing the first sentence to something like this: `Source components are named collections of directory filters.`

Maybe a link to the directory filter documentation would be nice here.